lib/snippets/publish-lerna-independent-packages

```diff
@@ -29,9 +29,9 @@ const packages = PackageUtilities.topologicallyBatchPackages(unsortedPackages, t
   .filter(({name}) => taggedPackages.includes(name));
 
 packages.forEach(({name, version}) => {
-  const command = `node_modules/.bin/lerna publish --yes --skip-npm=false --skip-git --force-publish=${name} --repo-version=${version} --scope=${name} --npm-tag=${npmTag(
-    version,
-  )}`;
+  const command = `node_modules/.bin/lerna exec --scope=${
```

Comment:
`lerna publish --npm-client=npm` should also work.

Reply:
Will this then up running `npm publish` instead `yarn publish`?

The [docs](https://github.com/lerna/lerna#--npm-client-client) may be outdated, but it seems that setting npm as the client will defer dependency installs to `npm install`.

Reply:
The flow is:
* [npm-client is read from the command line in `PublishCommand`](https://github.com/lerna/lerna/blob/2.x/src/commands/PublishCommand.js#L175)
* [config is passed to `publishTaggedInDir`](https://github.com/lerna/lerna/blob/2.x/src/commands/PublishCommand.js#L729)
* [client is used in an `exec` call](https://github.com/lerna/lerna/blob/a764945f7ebbac4e91dc9042b910085436daa55a/src/NpmUtilities.js#L155)

So I'm guessing it would work.  Haven't tried, though.

Reply:
Interesting! Maybe I'll put up a PR with that implementation, since Tzvi won't be here for the next couple days anyway.

Reply:
Ooo, excellent. 

It might be worth testing out `lerna publish --npm-client "npm" --registry "https://packages.shopify.io/shopify/node/npm/"`.

Reply:
Perfect! We should still look at pinning the lerna version, though, because the require tree changes too. They're switching to a monorepo afaict.

Reply:
In the future, can you document stuff like this in PR descriptions, please?